search.go

```diff
@@ -1,139 +1,79 @@
 package imdb
 
 import (
+	"encoding/json"
 	"errors"
 	"fmt"
-	"io/ioutil"
+	"io"
 	"net/http"
-	"net/url"
-	"regexp"
-	"strconv"
+	"strings"
 )
 
-const searchURL = "https://www.imdb.com/find?%s"
+const searchURL = "https://v3.sg.media-imdb.com/suggestion/x/%s.json?includeVideos=0"
```

Comment:
I'm slightly concerned about long-term stability of this, but I guess we'll see.

Reply:
I can say I am too, but I assume if they would want to break something they could break anything